### ISSUE TITLE
Accessibility - Filter button is not accessible with voice command as no aria-label has been provided

### ISSUE THREAD

**Issue by MariaAug** [microsoft-search/pnp-modern-search#1338]
**Version used**
4.2.2.0

**Describe the bug**
Filter button is not accessible with voice command using Dragon tool as no aria-label has been provided.

**To Reproduce**
1. Add PnP search box, search filter and search results web part on SharePoint page.
2. Configure the filter web part to show filters like Created, Modified.
2. Save the page.
3. Using Dragon tool, provide voice command as Filter. The filter icon will not be accessible.

**Expected behavior**
All the interactive elements must be accessible with voice command. The filter icon should be accessible when the voice command is provided as Filter.

**Screenshots**
![image](https://user-images.githubusercontent.com/90712499/134523632-520d2422-bbd3-4b67-832d-1c1db4c82a4d.png)
